framework/src/ble_manager/ble_manager_state.c

```diff
@@ -138,6 +138,21 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 
 		ret = ble_drv_delete_bond_all(mode);
 	} break;
+	case BLE_EVT_CMD_CONN_IS_ACTIVE: {
+		BLE_STATE_CHECK;
+
+		blemgr_msg_params *param = (blemgr_msg_params *)msg->param;
+		trble_conn_handle con_handle = *(trble_conn_handle *)param->param[0];
```

Comment:
Is it okay not to check `param` is NULL or not?

Reply:
Yes, It is guranteed.  This functions are not for general users.